testsuite/features/init_clients/allcli_update_activationkeys.feature

```diff
@@ -45,6 +45,7 @@ Feature: Update activation keys
     And I disable source package syncing
   
 @scc_credentials
+@susemanager
```

Comment:
Just to be on the safe side, even if for now we don't have GitHub validation in SUMA, I would also add the tag here.

Reply:
Sure. Fixed.